### PR TITLE
GitHub Actions: remove get-diff-action from preview

### PR DESCRIPTION
**Pull request by bershanskiy**
This Action does not use the diff, just the fact that at least one of three specific files changed, so it's possible to use only built-in scripting tool [`on.pull_request.paths`](https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#onpushpull_requestpaths).

### CODE REVIEW

File: .github/workflows/preview.yml [hunk 1]
```diff
@@ -2,6 +2,10 @@ name: Preview
 
 on:
   pull_request:
+    paths:
+    - package.json
+    - yarn.lock
+    - preview.yml
```
Comment (peterbe) [inline .github/workflows/preview.yml:8]: This makes me wonder, does it even work?
Screen Shot 2020-12-02 at 8 48 40 AM

Clearly the `on.pull_request.paths` seems to say that "No, nothing's changed so don't bother running". 
But you did change *this* file that we're looking at. 

```suggestion
    - .github/workflows/preview.yml
```
??

Comment (peterbe) [reply .github/workflows/preview.yml:8]: I'm skimming [the docs](https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#onpushpull_requestpaths) and it basically says nothing about it needing to be absolute :)
Also, in ["Filter pattern cheat sheet"](https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet) every single example is something that has some form of pattern like "*" or "!". 

But pretty sure this workflow should have run because there was a change to the workflow file itself.

Comment (bershanskiy) [reply .github/workflows/preview.yml:8]: I added included the workflow file in paths to always test every change to the workflow itself. This testing isn't necessary, especially considering how simple workflow is. It's just a precaution that avoids an edge case where workflow file is modified and merged without testing, and fails only later.

Consider the following repository (I just created to show this edge case):
https://github.com/bershanskiy/actions-test/

The commits (in chronological order) do the following:
1. GitHub Actions workflow is created, which runs only on change to `README.md` and _always_ fails when runs
    This commit does not trigger workflow run, so it "passes".
2. `NOT-README.md` is created, and does not trigger workflow, so it "passes".
     There can be multiple commits, which don't trigger workflow, all will "pass"
3. A while later, `README.md` is created and workflow is triggered for the first time, so this commit fails.

The point is: the bug was introduced in commit 1, but was not noticed until commit 3.

Comment (bershanskiy) [reply .github/workflows/preview.yml]: > I'm skimming the docs and it basically says nothing about it needing to be absolute :)

👍 Yes, but I incorporated your suggestion to change `preview.yml` to `.github/workflows/preview.yml` just in case.

**Review by peterbe — APPROVED** [review]

**Comment by peterbe** [comment]
THANK YOU @bershanskiy for your contribution! We very much appreciate it.
